src/components/http-error-message/components/test/e2e/http-error-message-preset.js

```diff
@@ -0,0 +1,26 @@
+import { VlElement, By } from '../../../../../utils/test';
```

Comment:
Een test folder in de components folder is wat raar vind ik, dan zou ik eerder deze losse file bij de e2e test folder van de http-error-message zetten. Ik dacht eerst dat het om een specifieke component van de components folder ging.